HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,48 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is the platform specific path separator.
+
+method::isVowel
+
+returns:: true if the char is a vowel.
+
+method::isControl
+
+returns:: true if the char is a control character.
+
+method::asAscii
+
+returns:: the object itself.
+
+method::asUnicode
+
+returns:: a single Unicode value. Ascii is a subset of Unicode.
```

Comment:
the function name doesn't make sense because Unicode is not a concrete encoding (encodings of Unicode include UTF-8, UTF-16, etc.). SC strings (and therefore characters) are implicitly UTF-8, but Char objects with values between 128 and 255 (or -128 and -1) will return values that aren't UTF-8 codepoints (they print as the <?> unknown character glyph).

I would just document that this returns "the object itself" like you did for asAscii and optionally document this discrepancy.

Reply:
ok. thanks.